### ISSUE TITLE
Get rid of some of the main* files

### ISSUE THREAD

**Issue by ssiccha** [jnkbrachter/GaussPar#8]
I guess a fully parallel and a fully sequential version would be enough. One probably doesn't even need two different files / functions for that.

There is some global variable in HPCGAP that one can check to differentiate HPCGAP from GAP. I think it's called something like `IsHPCGAP`?

**Comment by ssiccha** [comment]
- `main_par_trafo.g` and `main_semi_par_trafo.g` should go completely.
- some of the `main*` files contain identical versions of a `ClearUp` function.

In addition also `read.g` and `read_hpc.g` can be merged into one file. Use `IsBound(IsHPCGAP)` to check whether GAP or HPC-GAP is loaded.

**Comment by ssiccha** [comment]
Once #6 is done, I hope the `diff`s between `main_full_par_trafo.g` and `main_seq_trafo.g` should be relatively small. If that is the case, getting rid of the two `main..` file should be easy.
